src/scrabble-long.js

```diff
@@ -0,0 +1,134 @@
+class Scrabble {
+  constructor (word) {
+    this.word = word
+    this.points = 0
+  }
+
+  // first find out if the string is empty string, whitepsace or null, then score 0.
+  score () {
+    if (
+      this.word === '' ||
+      this.word === ' ' ||
+      this.word === ' \t\n' ||
+      this.word === null
+    ) {
+      this.points += 0
+      return 0
+    }
+
+    // I use for loop on the string to score each letter on each iteration
+    for (let i = 0; i < this.word.length; i++) {
+      // thise are the letters in the corresponting arrays (based on the points value), stored in a variable
+      const pointOne = ['a', 'e', 'i', 'o', 'u', 'l', 'n', 'r', 's', 't']
+      const pointTwo = ['d', 'g']
+      const pointThree = ['b', 'c', 'm', 'p']
+      const pointFour = ['f', 'h', 'v', 'w', 'y']
+      const pointFive = ['k']
+      const pointEight = ['j', 'x']
+      const pointTen = ['q', 'z']
+      // i declared variables for the double/triple points, it can be either True or False
+      let doublePoints = false
+      let triplePoints = false
+      // if the actual character in the string in this actual iteration of the loop is "{" or "[" => then double points will switch to True AND i++ so we jump on the next letter in the string and use IF statements to find out if that letter is TRUE, FALSE or else.
+      if (this.word[i] === '{') {
+        doublePoints = true
+        i++
+      }
+      // if the character is "}" or "]" then double points will swithc to FALSE therefore the letter after this character only worth single points.
+      if (this.word[i] === '}') {
+        doublePoints = false
+      }
+
+      if (this.word[i] === '[') {
+        triplePoints = true
+        i++
+      }
+
+      if (this.word[i] === ']') {
+        triplePoints = false
+      }
+
+      if (
+        pointOne.includes(this.word[i]) ||
+        pointOne.includes(this.word[i].toLowerCase())
```

Comment:
theres quite a lot of repetition of `.toLowerCase()` here, potentially could you convert this.word to lowercase before you enter it into the for loop? e.g `this.word = this.word.toLowerCase();` on line 50 and then take out all the toLowerCase lines?